boms/cloud-oss-bom/RELEASING.md

```diff
@@ -62,11 +62,12 @@ new release is available on Maven Central.
 * Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
     * Go to go/java-live
     * Sort by title
-    * Scroll down until you find the section with titles "chore(deps): update dependency com.google.cloud:libraries-bom to v<version>"
-   * Approve and merge these PRs.
+    * Scroll down until you find the section with titles "chore(deps): update dependency com.google.cloud:libraries-bom to v&lt;version>"
+    * Approve and merge these PRs.
+      For PRs waiting for Kokoro jobs, add the "kokoro:force-run" label.
```

Comment:
I had to add this label to https://github.com/googleapis/java-errorreporting/pull/291#issuecomment-720556541